auth_oauth_enforced/readme/CONTEXT.md

```diff
@@ -0,0 +1,3 @@
+For example, some companies may have wanted to enforce OAuth for their users.
+On the other hand, there may be those who prefer not to enforce OAuth for users
+such as stakeholders accessing the portal.
```

Comment:
```suggestion
Sometimes, you may want to enforce OAuth login for your employees for security reasons,
while it may not be appropriate to do so for portal users. This module intends to
address this need.
```